docs/developer/contributing/running-kibana-with-apm.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[running-kibana-with-apm]]
+== Running Kibana with the APM Agent Locally
+
+The easiest and recommended way of running Kibana with the APM agent locally is to use the solution provided by the https://github.com/elastic/apm-integration-testing[apm-integration-testing] repo. You’ll need https://www.docker.com/community-edition[Docker] and https://docs.docker.com/compose/install/[Docker Compose] to use the tool.
```

Comment:
I think it would be worth adding a paragraph here about why a developer would want to do this with examples of the types of information that can be captured with the APM agent currently.

Reply:
Is [this](https://github.com/elastic/kibana/pull/84700/commits/886e586b089f20c54068f9a3192fb8bdf5a7dac4#diff-7e06444f575533e2af08b335d5dbf4a78b03eb69c2ef3f2f9f0ee286442248e4R4-R10) ok?